views/index.erb

```diff
@@ -43,6 +43,19 @@
     <div class="jumbotron jumbotron-skinny"></div>
 
     <main class="container mb-5 mt-5">
+      <div class="row text-center">
+        <div class="col-sm-6">
+          <h5 class="text-success">Total Pending Requests</h5>
+          <h4 class="text-success"><%= summary_data[:active] %></h4>
+        </div>
+        <div class="col-sm-6">
+          <h5 class="text-success">Requests in Progress</h5>
+          <h4 class="text-success"><%=summary_data[:in_progress] %></h4>
+        </div>
+      </div>
+      <br>
```

Comment:
Only thing id say is remove this `br` tags and use css padding or margin.

Reply:
resolved in https://github.com/madetech/requestinator/pull/12/commits/7fea29a1e7dcd0ee97e29fb45a84374e0f2c0d5c